README.md

```diff
@@ -9,9 +9,14 @@
 To build and run the container locally with hot reload on python files do:
 ```
 DOCKER_BUILDKIT=1 docker build . -t gbnc
-docker run  -v "$(pwd)/gswikichat":/workspace/gswikichat \
-            -p 8000:8000 --rm --name gbnc -it gbnc \
-            -e HUGGING_FACE_HUB_TOKEN=$HUGGING_FACE_HUB_TOKEN
+docker run  \
+  -v "$(pwd)/gswikichat":/workspace/gswikichat \
+  -v "$(pwd)/cache":/root/.cache \
+  -e HUGGING_FACE_HUB_TOKEN=$HUGGING_FACE_HUB_TOKEN
```

Comment:
I initially tried this with `-e HUGGING_FACE_HUB_TOKEN=$HUGGING_FACE_HUB_TOKEN` on the command line with `docker run`. 
Unfortunately, I do not know why, but the `-e` seems to conflict with the `-v` or another flag.

As such, I felt forced to 
1. Use `--env` instead of `-e` when assigning env variables
2. Place the `--env  HUGGING_FACE_HUB_TOKEN=$HUGGING_FACE_HUB_TOKEN` at the top of the stack of docker run flags.

Reply:
Ah, I see. Interesting observation.

At the moment we technically do not need a huggingface token, as the embedding model we download is accessible without login. Maybe we can even remove the token from the example command again.

Reply:
fix in the docs